numpy/core/numeric.py

```diff
@@ -1120,7 +1212,6 @@ def restoredot():
     alterdot : `restoredot` undoes the effects of `alterdot`.
 
     """
-    # 2014-08-13, 1.10
```

Comment:
i think they were removed in another commit from the master branch that was merged into this.